process-migration-service/frontend/package.json

```diff
@@ -13,7 +13,8 @@
     "start": "node src/bin/server.js",
     "test": "jest",
     "precommit": "lint-staged",
-    "locktt": "locktt"
+    "locktt": "locktt",
+    "test:it": "cypress run 9000"
```

Comment:
we've removed the part where the app is actually being started before the tests .... please put back the `start-server-with-test` script. Also please make sure that the initial test spec tries to access the exposed port to verify we do things right.

Reply:
ok